lib/presentation/widget/upload_request_group/upload_request_group_viewmodel.dart

```diff
@@ -58,6 +58,8 @@ class UploadRequestGroupViewModel extends BaseViewModel {
   List<UploadRequestGroup> _uploadRequestListActiveClosed = [];
   List<UploadRequestGroup> _uploadRequestListArchived = [];
 
+  int _tabIndex = 0;
```

Comment:
why we need store here? How about store it directly in UI state?

Reply:
I thought dispatching an action at every tab switch was a bad idea and useless wdyt ?

Reply:
yes, it should be consistent for all app. Not too many places to store the state of the application.